transcriber.py

```diff
@@ -123,7 +123,7 @@ def transcribe_callback(_):
             if (not audio_model or not loaded_audio_model) or ((audio_model and loaded_audio_model) and loaded_audio_model != model):
                 device = 'cpu'
                 if torch.has_cuda:
-                    device = 0
+                    device = "cuda"
```

Comment:
Should this be "cuda:0"?
I suppose if you just put "cuda" it defaults to a device?

Reply:
Looked it up, this works.